xonsh/completer.py

```diff
@@ -7,12 +7,12 @@
 import subprocess
 import sys
 
+from builtins import __xonsh_env__ as ENV
```

Comment:
The probably with importing globally like this is that it prevents people from swapping out `__xonsh_env__` wholesale and still having everything work.  The other instances of ENV should probably be removed, too, but that is for another PR :)

`GET_ENV` would be fine, but since it is a function it should probably be `get_env` .